themes/doom-dracula-theme.el

```diff
@@ -237,6 +237,29 @@ determine the exact padding."
    (web-mode-string-face :foreground yellow)
    (web-mode-type-face :foreground violet)
 
+   ;; helm
+   (helm-bookmark-w3m :foreground violet)
+   (helm-buffer-not-saved :foreground violet)
+   (helm-buffer-process :foreground orange)
+   (helm-buffer-saved-out :foreground fg)
+   (helm-buffer-size :foreground fg)
+   (helm-candidate-number :foreground bg :background fg)
+   (helm-ff-directory :foreground green :weight 'bold)
+   (helm-ff-executable :foreground dark-blue :inherit 'italic)
+   (helm-ff-invalid-symlink :foreground magenta :weight 'bold)
+   (helm-ff-prefix :foreground bg :background magenta)
+   (helm-ff-symlink :foreground magenta :weight 'bold)
+   (helm-grep-finish :foreground base2)
+   (helm-grep-running :foreground green)
+   (helm-header :foreground base2 :underline nil :box nil)
```

Comment:
Could you restore the background of `helm-source-header`? It is intended to make it more distinct from the other entries.

```diff
-(helm-source-header :foreground base2 :underline nil :box nil)
+((helm-source-header &override) :foreground base2 :underline nil :box nil)
```

The `&override` will cause it to inherit the properties from the base theme.

Reply:
Thanks, I changed it to only override the `foreground`.